bundles/org.openhab.binding.philipsair/src/main/java/org/openhab/binding/philipsair/internal/model/PhilipsAirPurifierDevice.java

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.philipsair.internal.model;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * Contains info details of the Air Purifier thing
+ *
+ * @author Michał Boroński - Initial contribution
+ *
+ */
+public class PhilipsAirPurifierDevice {

```

Comment:
To deal with the `@NonNullByDefault` warnings you can do one of three things:
1. Add the `@NonNullByDefault` to the class.
2. Make sure that your class name ends in DTO
3. Make sure that your class is in a package or subpackage called `dto`.

Please do one of the three.